File: matiasmasso/Mat.NET/DT2/DTOCliProductDto.vb
Public Class DTOCliProductDto
    Inherits DTOBaseGuid

    Property Customer As DTOCustomer
    Property Product As DTOProduct
    Property Dto As Decimal

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub
End Class

File: matiasmasso/Mat.NET/DTO/DTOCliProductDto.cs
using System;

namespace DTO
{
    public class DTOCliProductDto : DTOBaseGuid
    {
        public DTOCustomer Customer { get; set; }
        public DTOProduct Product { get; set; }
        public decimal Dto { get; set; }

        public DTOCliProductDto() : base()
        {
        }

        public DTOCliProductDto(Guid oGuid) : base(oGuid)
        {
        }
    }
}
